tests/functional/labels.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'require'
+], function (intern, registerSuite, assert, require) {
+  'use strict';
+
+  var url = function (num) {
+    return intern.config.siteRoot + '/issues/' + num;
+  };
+
+  registerSuite({
+    name: 'labels',
+
+    'label gear is visible': function () {
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url(100)))
+        .sleep(10000)
+        .findByCssSelector('.LabelEditor-launcher span.wc-sronly')
+        .getVisibleText()
+        .then(function (text) {
+          assert.strictEqual(text, 'Edit Labels', 'The label gear text is available');
+        })
+        .end();
+    }
+
+    // 'setting a label': function () {
```

Comment:
Do not **merge** this in master. :) It's not expected to be merged yet. I'm still trying to develop a test which will be working. It's just on a PR because you asked for it. :)